sql/hive/src/test/scala/org/apache/spark/sql/HiveCharVarcharTestSuite.scala

```diff
@@ -50,6 +51,35 @@ class HiveCharVarcharTestSuite extends CharVarcharTestSuite with TestHiveSinglet
       assert(rest.contains("CHAR(5)"))
     }
   }
+
+  // TODO(SPARK-34203): Move this too super class when the ticket gets fixed
```

Comment:
`too` -> `to`

Reply:
BTW the jira is about null partition value, while this test doesn't cover null partition at all.

Reply:
I rechecked this, this is a valid case. moved to super. The null case seems an unrelated issue and need not to add a TODO here